src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
-import React from 'react';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../react-app-env';
+
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => (
+      <div className="card" key={movie.imdbId}>
```

Comment:
```suggestion
  <ul className="movies">
    {movies.map(movie => (
      <li className="card" key={movie.imdbId}>
```
Краще використовувати семантичні теги. Фактично, це в тебе список, тому ul-li було б дуже доречним